### PR TITLE
httpie 1.0.3

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by javian** [comment]
Awaiting further updates before merging.

**Comment by jeffbyrnes** [comment]
FYI, upstream changes were accepted by the httpie maintainer.

**Comment by chenrui333** [comment]
Cool, thanks!

**Review by jeffbyrnes — CHANGES_REQUESTED** [review]

File: Formula/httpie.rb [hunk 1]
```diff
@@ -61,3 +63,66 @@ def install
...
+   resource "idna" do
+@@ -53,8 +53,8 @@ class Httpie < Formula
+   end
+
+   resource "PySocks" do
+-    url "https://files.pythonhosted.org/packages/53/12/6bf1d764f128636cef7408e8156b7235b150ea31650d0260969215bb8e7d/PySocks-1.6.8.tar.gz"
+-    sha256 "3fe52c55890a248676fd69dc9e3c4e811718b777834bcaab7a8125cf9deac672"
++    url "https://files.pythonhosted.org/packages/15/ab/35824cfdee1aac662e3298275fa1e6cbedb52126d1785f8977959b769ccf/PySocks-1.7.0.tar.gz"
++    sha256 "d9031ea45fdfacbe59a99273e9f0448ddb33c1580fe3831c1b09557c5718977c"
+   end
```
[the remark this answers is not in the document]
Comment (SMillerDev) [reply Formula/httpie.rb]: Probably even easier to just use the url of the upstream patch. For example: https://github.com/Homebrew/homebrew-core/blob/master/Formula/upx.rb#L21-L24

[the remark this answers is not in the document]
Comment (chenrui333) [reply Formula/httpie.rb]: Updated with using the online commit hash.

**Comment by SMillerDev** [comment]
Why does it need to patch a file that generates the formula in the formula? It won't affect any usage of httpie right?

**Comment by javian** [comment]
There's a script that can extract the dependencies https://pypi.python.org/pypi/homebrew-pypi-poet which are then inserted into the formula and this is what needs to be done here and all other formulae constructed this way.

**Comment by jeffbyrnes** [comment]
So yeah, the patch won’t work that way, since this is a generated binary we’re installing.

So either @javian’s suggestion, or manually copying in the new reps, is what’s needed here.

@chenrui333 if you want to close this, I’m happy to take care of it instead!

**Comment by fxcoudert** [comment]
@jeffbyrnes we'll take you up on that offer :)

**Pull request closed without merging**